docs/UserGuide.md

```diff
@@ -189,10 +192,18 @@ Examples:
 
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all entries from the address book or clears all contacts with the specified tags.
```

Comment:
Maybe this can be phrased as "clears all entries from the address book **if tags are not specified**..."